thentos-core/src/Thentos.hs

```diff
@@ -71,8 +71,6 @@ main = makeMain $ \ actionState mBeConfig mFeConfig cmd ->
 
             void $ concurrently backend frontend
 
-        RunSso -> error "RunSso: not implemented."
```

Comment:
Can we just remove `cmd`/`Command` altogether? It doesn't seem to be serving a purpose anymore.